central/policy/datastore/datastore_impl_postgres_test.go

```diff
@@ -76,8 +77,8 @@ func (s *PolicyPostgresDataStoreTestSuite) SetupTest() {
 
 	s.categoryDS = policyCategoryDS.New(categoryStorage, categorySearcher, policyCategoryEdgeDS.New(edgeStorage, edgeSearcher))
 
-	policyStore := pgStore.CreateTableAndNewStore(s.ctx, s.db, s.gormDB)
-	s.datastore = New(policyStore, search.New(policyStore), s.mockClusterDS, s.mockNotifierDS, s.categoryDS)
+	policyDS := policyStore.New(s.db)
```

Comment:
nit:  This is still a store vs a datastore.  I think adding the DS here implies a datastore which would not be correct.